specification/cognitiveservices/data-plane/TranslatorText/Readme.md

```diff
@@ -0,0 +1,4 @@
+# Microsoft Translator API V3
```

Comment:
You'll need to rename this file to something else as ```readme.md``` has special meaning in this repo.  Or just remove it entirely as I don't know how much value it provides regarding swagger generation.